tools/circleci_dependencies.sh

```diff
@@ -20,7 +20,6 @@ if [[ "$CIRCLE_JOB" == "interactive_test" ]]; then
 	python -m pip install --progress-bar off --upgrade --pre --only-binary ":all:" numba llvmlite
 	wget -q https://osf.io/kej3v/download -O vtk-9.0.20201117-cp39-cp39-linux_x86_64.whl
 	python -m pip install --progress-bar off vtk-9.0.20201117-cp39-cp39-linux_x86_64.whl
-	python -m pip install --progress-bar off https://github.com/pyvista/pyvista/zipball/master
```

Comment:
I went with a solution that should be slightly easier to eventually revert in https://github.com/mne-tools/mne-python/pull/9284/files#diff-838a9c60bcbbc8d01b84e4169048fe3fc03667b383b463c501857b168e54d082R13

Reply:
Okay sure, I'll use that then